controller/Publish.php

```diff
@@ -34,26 +36,26 @@
  * @license GPL-2.0
  *
  */
-class Publish extends \tao_actions_CommonModule {
+class Publish extends \tao_actions_SaSModule {
 
     use OntologyAwareTrait;
     
-    public function wizard()
+    public function wizard(): void
```

Comment:
Also, Can you check where this action is used? I'm not sure that this code is used anywhere.